packages/components/table/src/composables/use-scrollbar.ts

```diff
@@ -0,0 +1,28 @@
+import { ref } from 'vue'
+import { isNumber } from '@element-plus/utils'
+
+export const useScrollbar = () => {
+  const scrollBarRef = ref()
+
+  const scrollTo = ({ top, left }: { top?: number; left?: number }) => {
+    setScrollTop(top)
+    setScrollLeft(left)
```

Comment:
**Suggestion:** let's use `element.scrollTo(options)` to implement this method.

Ref: https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollTo